packages/tools/src/lib/build-cache/common.ts

```diff
@@ -10,28 +10,79 @@ export type SupportedRemoteCacheProviders = 'github-actions';
 
 export type RemoteArtifact = {
   name: string;
-  downloadUrl: string;
+  url: string;
+  id?: string;
```

Comment:
maybe it would be better to get rid of name and only have `id: string, url: string`.

if name is for display and id is for identifying then i'd expect id to be required and name to be optional - and current providers that use name for identifying can just call it id instead.

Reply:
So in this case `id` is not really something that we're using. Name is essential as this is how we're identifying an artifact based on platform, build variant and fingerprint. I'll remove id to avoid confusion